pallets/pool-registry/src/tests.rs

```diff
@@ -75,6 +75,7 @@ fn register_pool_and_set_metadata() {
 				currency,
 				max_reserve,
 				metadata.clone(),
+				BoundedVec::default()
```

Comment:
I would add a test for checking they are added validly here.

Reply:
Ticket created: https://github.com/centrifuge/centrifuge-chain/issues/1499